plugins/modules/gitlab_group_access_token.py

```diff
@@ -0,0 +1,322 @@
+
+options:
+  group:
+    description:
+      - ID or full path of group in the form of group/subgroup.
+    required: true
+    type: str
+  name:
+    description:
+      - Access token's name.
+    required: true
+    type: str
+  scopes:
+    description:
+      - Scope of the access token.
+    required: true
+    type: list
+    elements: str
+    aliases: ["scope"]
+    choices: ["api", "read_api", "read_registry", "write_registry", "read_repository", "write_repository", "create_runner", "ai_features", "k8s_proxy"]
+  access_level:
+    description:
+      - Access level of the access token.
+    type: str
+    default: maintainer
+    choices: ["guest", "reporter", "developer", "maintainer", "owner"]
+  expires_at:
+    description:
+      - Expiration date of the access token in YYYY-MM-DD format.
+    type: str
+    required: true
+  recreate:
+    description:
+      - Whether the access token will be recreated if there is a difference between desired state and actial state.
+    type: bool
+    default: false
+  state:
+    description:
+      - When V(present) the access token will be added to the group if it does not exist.
+      - When V(absent) it will be removed from the group if it exists.
+    default: present
+    type: str
+    choices: [ "present", "absent" ]
+'''
+
+EXAMPLES = r'''
+- name: "Creating a group access token"
+  community.general.gitlab_group_access_token:
+    api_url: https://gitlab.example.com/
+    api_token: "somegitlabapitoken"
+    group: "my_group/my_subgroup"
+    name: "group_token"
+    expires_at: "2024-12-31"
+    access_level: developer
+    scopes:
+      - api
+      - read_api
+      - read_repository
+      - write_repository
+    state: present
+
+- name: "Revoking a group access token"
+  community.general.gitlab_group_access_token:
+    api_url: https://gitlab.example.com/
+    api_token: "somegitlabapitoken"
+    group: "my_group/my_group"
+    name: "group_token"
+    expires_at: "2024-12-31"
+    scopes:
+      - api
+      - read_api
+      - read_repository
+      - write_repository
+    state: absent
+
+- name: "Change (recreate) existing key (used for extending tokens validity, or changing its parameters by reissuing it)"
+  community.general.gitlab_group_access_token:
+    api_url: https://gitlab.example.com/
+    api_token: "somegitlabapitoken"
+    group: "my_group/my_group"
+    name: "group_token"
+    expires_at: "2024-12-31"
+    scopes:
+      - api
+      - read_api
+      - read_repository
+      - write_repository
+    recreate: true
+    state: present
+'''
+
+RETURN = r'''
+msg:
+  description: Success or failure message.
+  returned: always
+  type: str
+  sample: "Success"
+
+result:
+  description: JSON parsed response from the server.
+  returned: always
+  type: dict
+
+error:
+  description: the error message returned by the GitLab API.
+  returned: failed
+  type: str
+  sample: "400: key is already in use"
+
+acecss_token:
+  description: API object.
```

Comment:
Shouldn't this mention:
```suggestion
  description:
    - API object.
    - Only contains the value of the token if the token was created or recreated.
```